src/test/java/org/jenkinsci/plugins/github/config/GitHubServerConfigIntegrationTest.java

```diff
@@ -14,10 +14,9 @@
 import net.sf.json.JSONObject;
 import org.eclipse.jetty.server.Server;
 import org.eclipse.jetty.server.ServerConnector;
-import org.eclipse.jetty.servlet.DefaultServlet;
-import org.eclipse.jetty.servlet.ServletContextHandler;
-import org.eclipse.jetty.servlet.ServletContextHandler.Context;
-import org.eclipse.jetty.servlet.ServletHolder;
+import org.eclipse.jetty.ee8.servlet.DefaultServlet;
+import org.eclipse.jetty.ee8.servlet.ServletContextHandler;
+import org.eclipse.jetty.ee8.servlet.ServletHolder;
```

Comment:
Jetty 11 and older were tied to a particular Servlet API version, but Jetty 12 and newer features a modular architecture that allow Jetty to be used with any Servlet API version. For EE 8, the classes have now been moved to these packages.

This is needed for this plugin's test suite to work against Jetty 12 EE 8, which PCT will start running after the 2.472 weekly release on Tuesday, August 13.